server/stycoon/tick.go

```diff
@@ -19,13 +19,13 @@ func (game *Game) getGameTick() {
 	}
 }
 
-func (game *Game) getTickWait() time.Duration {
+func (game *Game) getTickLeftMs() time.Duration {
```

Comment:
Let's rename this to getTickLeft

Reply:
getRemainingTime
getTimeLeft

Reply:
`getRemainingTickTime` thanks that is much better ;)